lib/src/neu/verifications/mismatches.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Mismatches are used by the contract mismatcher to report on mismatches
+ * between data and different components of a contract. They are not intended
+ * to produce human friendly messages as they may not contain a sufficient
+ * amount of contextual information.
+ */
+
+export enum MismatchKind {
+  REQUIRED_HEADER_MISSING = "required_header_missing",
+  UNDEFINED_HEADER = "undefined_header",
+  HEADER_TYPE_DISPARITY = "header_type_disparity",
+  PATH_PARAM_TYPE_DISPARITY = "path_param_type_disparity",
+  REQUIRED_QUERY_PARAM_MISSING = "required_query_param_missing",
+  UNDEFINED_QUERY_PARAM = "undefined_query_param",
+  QUERY_PARAM_TYPE_DISPARITY = "query_param_type_disparity",
+  UNDEFINED_BODY = "undefined_body",
+  BODY_TYPE_DISPARITY = "body_type_disparity"
+}
+
+export interface RequiredHeaderMissingMismatch {
```

Comment:
I saw you renamed `Mismatch` to `DisparityViolation`, should the interfaces in this file also be renamed?

Look the comment below for an example👇

Reply:
explained offline, I didn't saw the comment on the file